### PR TITLE
tests for passing accession flag to /resource, rubocop touchups

### PR DESCRIPTION
**Pull request by jmartin-sul**
pass as URL param, as sdr-client does.

## Why was this change made?

show that the `accession` parameter is passed along correctly to `IngestJob` by the `/resources` route.

see also https://github.com/sul-dlss/sdr-client/pull/148

## How was this change tested?

tests only

## Which documentation and/or configurations were updated?

n/a

### CODE REVIEW

**Comment by jmartin-sul** [comment]
see also see also sul-dlss/happy-heron#1191 and https://github.com/sul-dlss/sdr-client/pull/148

File: spec/requests/create_resource_spec.rb [hunk 1]
```diff
@@ -141,9 +141,10 @@
       file_params['size'] = 10
       model_params
     end
+    let(:path) { '/v1/resources' }
```
Comment (jcoyne) [inline spec/requests/create_resource_spec.rb:144]: I find that putting this as a variable is harder to read than how it was before.  This isn't used more than one time, so why?

Comment (jmartin-sul) [reply spec/requests/create_resource_spec.rb]: was cruft from a slightly different implementation, moved back inline

**Comment by jmartin-sul** [comment]
suggestion taken, threw in a bit of rubocop while i was at it

**Review by jcoyne — APPROVED** [review]

**Pull request merged**